src/Stack/Ide.hs

```diff
@@ -43,7 +43,7 @@ ide
     -> [String] -- ^ GHC options.
     -> m ()
 ide targets useropts = do
-    (_realTargets,_,pkgs) <- ghciSetup Nothing targets
+    (_realTargets,_,pkgs) <- ghciSetup (Just BSAll) Nothing targets
```

Comment:
The new behavior of continuing is better, I think.  At least that way the user ends up with a GHCI session that's somewhat setup.